salt/preflight/mandatory.sls

```diff
@@ -0,0 +1,45 @@
+{% import_yaml 'preflight/defaults.yaml' as settings %}
+
+install mandatory packages:
+  pkg.installed:
+    - pkgs: {{ settings.mandatory.packages }}
+
+{%- for directory in settings.mandatory.available_pathdirs %}
+make sure {{ directory }} does not exist:
+  file.absent:
+    - name: {{ directory }}
+{%- endfor %}
+
+{%- for service in settings.mandatory.services.running %}
+enable {{ service  }} service:
+  service.running:
+    - name: {{ service  }}
+    - enable: true
+{%- endfor %}
+
+add the module br_netfilter to kernel:    # required to set sysctl net.bridge.bridge-nf-call-iptables
```

Comment:
line too long, put the comment on the line above

Reply:
Intent is to put some linting tools in place (@gdemonet @jbertran) to enforce 'rules' on code/YAML/... As long as those are not in place, let's not spend too much time enforcing them through peer review, and fix things up once the linters are running.

However, valid remark, and given the need for other changes I guess @ramanduh can apply a fix for this one easily :smiley:

Reply:
Agreed.

On the other hand, if we wait for the linter then we will have a big commit "fix formatting" that will "pollute" the git blame.

As a middle ground we can decide to only treat formatting-related remark if there is also a need for non-formatting changes. (i.e. formatting-only review should be avoided)

Reply:
We could also include such a linting Eve target for the 2.0.0-alpha1 milestone